stage #3/java/module #2. REST API Basics/task02/repository/src/test/resources/DBScript.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE gift_certificate
+(
+    gift_certificate_id  INT  NOT NULL PRIMARY KEY IDENTITY,
+    name             VARCHAR(255)    NOT NULL,
+    description      VARCHAR(300)      NOT NULL,
+    price            DECIMAL(10, 2)  NOT NULL,
+    duration         INT       NOT NULL,
+    create_date      DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    last_update_date DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    remove           BOOLEAN  DEFAULT FALSE     NOT NULL
+);
+
+CREATE TABLE tag
+(
+    tag_id   INT  NOT NULL PRIMARY KEY IDENTITY,
+    name    VARCHAR(255)           NOT NULL,
+    remove BOOLEAN DEFAULT FALSE NOT NULL
+);
+
+CREATE TABLE tag_gift_certificate
+(
+    gift_certificate_id INT,
+    tag_id              INT,
+    PRIMARY KEY (gift_certificate_id, tag_id),
+    FOREIGN KEY (gift_certificate_id) REFERENCES gift_certificate (gift_certificate_id),
+    FOREIGN KEY (tag_id) REFERENCES tag (tag_id)
+);
+
+
+INSERT INTO tag (name) VALUES('Extreme');
+INSERT INTO tag (name) VALUES('For him and for her');
+INSERT INTO tag (name) VALUES('Training');
+INSERT INTO tag (name) VALUES('Flying');
+INSERT INTO tag (name) VALUES('In another country');
+INSERT INTO tag (name) VALUES('Ride and Walk');
+
+INSERT INTO gift_certificate (name,description,price,duration,create_date,last_update_date) VALUES
```

Comment:
That's good) You have Script for creation the schema of your DB. Do you think anyone needs your fake data?